### PR TITLE
[no-relnote] Bump to Holodeck V0.2.3

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Let's merge https://github.com/NVIDIA/k8s-device-plugin/pull/891 first

File: .github/workflows/staging_e2e.yaml [hunk 1]
```diff
@@ -36,12 +36,12 @@ jobs:
           echo "COMMIT_SHORT_SHA=${GITHUB_SHA:0:8}" >> $GITHUB_ENV
           GOLANG_VERSION=$(./hack/golang-version.sh)
           echo "GOLANG_VERSION=${GOLANG_VERSION##GOLANG_VERSION := }" >> $GITHUB_ENV
-          echo "HOLODECK_VCENTER_USERNAME=${HOLODECK_VCENTER_USERNAME}" >> $GITHUB_ENV
-          echo "HOLODECK_VCENTER_PASSWORD=${HOLODECK_VCENTER_PASSWORD}" >> $GITHUB_ENV
-          cat /home/runner/.ssh/vsphere-key > ${GITHUB_WORKSPACE}/holodeck_ssh_key
       - name: Set up Holodeck
-        uses: NVIDIA/holodeck@v0.2.2
```
Comment (elezar) [inline .github/workflows/staging_e2e.yaml:43]: Should this be done for the e2e action as in #914 too?

Comment (ArangoGutierrez) [reply .github/workflows/staging_e2e.yaml:43]: Yes, I'll edit

Comment (ArangoGutierrez) [reply .github/workflows/staging_e2e.yaml:43]: Done

File: .github/workflows/staging_e2e.yaml [hunk 2]
```diff
@@ -36,12 +36,12 @@ jobs:
...
+        uses: NVIDIA/holodeck@v0.2.3
         with:
+          vsphere_username: ${{ secrets.VSPHERE_USERNAME }}
```
Comment (elezar) [inline .github/workflows/staging_e2e.yaml:42]: Just so it's clear: This means that v0.2.3 of holodeck includes `INPUT_` variables for all these values?

**Review by elezar — APPROVED** [review]

**Pull request merged**